Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/ammo.yml

```diff
@@ -9,6 +9,8 @@
     WeaponRifleNovaliteC1: 4294967295 # Frontier: Infinite
     WeaponShotgunDoubleBarreled: 4294967295 # Frontier: 10<Infinite
     WeaponRevolverArgenti: 4294967295 # Frontier: 10<Infinite
+    MagazineBox454Casull:  4294967295 # Frontier
+    MagazineBox410cal: 4294967295 # Frontier
```

Comment:
I don't believe those ammo types should be available from a vendomat, especially given that the only gun that uses them isn't as easily obtainable. Can be a rare(ish) ammo crate find on expeds, or maybe a recipe for ammo fab.

Reply:
There will be another 410 gun in the future just wanted to do 1 gun at a time. I could add the other 410 here If you want. 

There will also be a rare 454casull in the future also